examples/v7-next/routes.js

```diff
@@ -0,0 +1,42 @@
+// This file was automatically added by edgio deploy.
+// You should commit this file to source control.
+import { Router } from '@edgio/core/router'
+import { nextRoutes } from '@edgio/next'
+
+// Remove this line to suppress Next's default behavior of removing trailing slashes via a redirect.
+// If trailingSlash: true is set in next.config.js, removing this line will remove the redirect that adds the trailing slash.
+nextRoutes.setEnforceTrailingSlash(true)
+
+export default new Router()
+  // NextRoutes automatically adds routes for all Next.js pages and their assets
+  .use(nextRoutes)
+  .match('/l0-api/:path*', ({ cache, proxy, removeUpstreamResponseHeader, removeRequestHeader }) => {
+    removeRequestHeader('+x-edg-serverless-hint')
```

Comment:
What's this used for? This header is not something that we should be setting from my understanding, and we don't document the usage of this.

Reply:
We need this as nextRoutes adds a header to all the paths as pointing to serverless, while this is not meant to go there. Everything inside nextRoutes is outside of the user's hand, hence removing the serverless hint header is the right way to go about it.

Reply:
Documented at https://docs.edg.io/guides/v7/performance/cdn_as_code#alter-requests-and-responses:~:text=headers%3A%20%7B%20set_request_headers%3A%20%7B%20%22x%2Dedg%2Dserverless%2Dhint%22%3A%20%22%22%20%7D%20%7D%2C

Reply:
Hmm, the example in the docs there is a bad example as that was automatically generated internally by the router for static assets. It made it out into the guide before I knew it was an internal header that shouldn't be exposed. I'll reach out to the team about this.

Reply:
Yeah I think DX can be looped in to ask for an ideal solution.